index.html

```diff
@@ -279,7 +279,14 @@ <h3>JsonSchemaCredential</h3>
                of a verifiable credential is <code>https://www.w3.org/ns/credentials#JsonSchema</code>.
             </p>
             <p>
-               The specification version of [[JSON-Schema]] can be any version noted in the section
+               When defining a credential schema using the <code>JsonSchemaCredential</code> type,
+               the credential's <code>credentialSubject</code> MUST use the <a href="#jsonschema">JsonSchema</a> 
```

Comment:
I don't understand this, are you saying the credential should have the following?

```json
{
  "credentialSubject": {
    "id": "my_fun_id",
    "type": "JsonSchema",
    "jsonSchema": {
      "$id": "my_fun_id"
    }
  }
}
```

Reply:
yes or 

```json
{
  "credentialSubject": {
    "type": "JsonSchema",
    "jsonSchema": {
      "$id": "my_fun_id"
    }
  }
}
```

Reply:
Ok that makes sense, but I couldn't find the place where it was "exemplified below". Did I miss anything?

Reply:
the existing examples already follow this guidance, but it was missing text to explain it